DEVELOPERS.md

```diff
@@ -62,6 +62,159 @@ running jobs from private GitHub repos, and/or you want to exercise the full tes
 suite (some tests in `tests/lib/test_git.py` are skipped if this environment variable
 is missing).
 
+## Operating principles
+
+The project retrieves jobs to be run from an [OpenSAFELY job
+server](https://github.com/opensafely-core/job-server) by polling the job server.
+
+Jobs belong to a `workspace`. This describes the git repo containing the
+OpenSAFELY-compliant project under execution; the git branch, and kind of
+database to use. The workspace also acts as a kind of namespace for
```

Comment:
Is "kind of database to use" still accurate?  I suppose there is still default vs t1oo?

Reply:
yeah, there is still default vs t1oo, I'm going to switch out "kind of" for "which" though